### DISCUSSION TITLE
Groesten Haus (Deimagined)

### DISCUSSION THREAD

**Discussion by framilano** [nzp-team/nzportable#674] [Map Releases]
## Groesten Haus (Deimagined)

[Download here](https://archive.org/details/fra_groesten)

### Description
A small survival based on Groesten Haus from WW2, **deimagined** because it's a reimagined demake :D

### Features

- Dogs
- Two little EEs
- All perks (including BO1 ones) and PaP

**Remember to report any bugs!**

<details>
<summary>Spoiler for EEs</summary>
To access power/mysterybox and the bunker containing perks and PaP you need to shoot the 3 teddies around the map:

- one is under the fridge on first floor
- another one is chilling on the window above the HIDE chalk, it's pretty high actually and maybe not that visible with low resolutions
- finally, the last one hidden behind the piano on second floor

To access the "hollow" wall you need to shoot all the explosive barrels outside of the map, the wall will move and it will grant you access to the Wunderwaffe DG2 wallbuy.
</details>

#### 06/03/2024
- Added EE music song
- Replaced some unseen textures with NULL

### Screenshots

![unknown_2024 03 02-16 01](https://github.com/nzp-team/nzportable/assets/28491164/4cecb35d-6ee1-491f-ab7a-b831eafed437)
![unknown_2024 03 02-16 12](https://github.com/nzp-team/nzportable/assets/28491164/158ae974-5b64-465f-accf-4d5c51127b6e)
![unknown_2024 03 02-16 12_1](https://github.com/nzp-team/nzportable/assets/28491164/bf069c76-c1ff-4598-9472-4d8fdf01d29c)
![unknown_2024 03 02-16 13_1](https://github.com/nzp-team/nzportable/assets/28491164/235e5f98-7125-44b0-ab70-070f82acbfb7)
![unknown_2024 03 02-16 13_2](https://github.com/nzp-team/nzportable/assets/28491164/3fb7f853-9e51-47de-ba90-e391884ba751)
![unknown_2024 03 02-16 16](https://github.com/nzp-team/nzportable/assets/28491164/f232f570-09a1-437e-a3b8-74459cd1d94c)
![unknown_2024 03 02-16 17](https://github.com/nzp-team/nzportable/assets/28491164/b653a9f1-7c54-4570-84c7-919854814fd4)
![unknown_2024 03 02-16 17_1](https://github.com/nzp-team/nzportable/assets/28491164/89678e80-51bd-4ea6-8312-0f9551f71d66)

**Comment by MotoLegacy** [comment]
Hi there, looks neat! Are you aware of the `scale` property for models? It allows you to change their sizing, so `resized_jeep.mdl` seems like an unneeded dependency :)

**Reply by framilano** [reply]
Fixing it immediately

**Update**: fixed!

**Comment by ImTander** [comment]
Fun map! I made it to round 8 on my first run. I only managed to find one teddy so those most be pretty well tucked away. I didn't however see any perks? I kinda have a pea brain so I wouldn't be surprised if you just have to do an ee to get them to show up.  couple side notes ; the room with the mystery box, doesn't have a buyable door to get in? I wasn't exactly to sure what to do. (again, pea brain) one last thing, which doesn't effect the game/map gameplay wise, is the weapons on the wall on the second floor are a little high up the wall. Interested to see more!

**Reply by framilano** [reply]
Glad you're liking it! Sure teddies are well hidden, if you manage to find them, every single doubt of yours will be solved 😉

Mmm regarding the wall weapons on the second floor they're placed at the same height we have at the first floor, weird, can you please send a screenshot and more details on the platform you're playing on?

**Reply by ImTander** [reply]
So it looks like I was mistaken about the wall weapons, they appear correctly. 

I did end up discovering the other teddies as well! I also put the map on my PS Vita an tested it there an it ran very well. I did run into a couple small bugs although I'm not sure if it was just my game or not. The first one being, after the dog round, the fog stayed at that thicker level of fog instead of going back to normal. The second, if you restart the map, it doesn't reset. The bunker and hollow will remain open if you opened them last match. 

Still, a very good map an I hope to be able to make maps with the same level of detail! Also sorry to be a bother, but I was wondering what you use to make maps? I've been trying to use WorldCraft but I seem to run into small things like being able to just walk through my perk machines. I tried trenchbroom but my wads don't load in the texture browser :( . Anyway sorry for all the typing and thanks for your time!

Images of the mentioned bugs : 
![IMG_3541](https://github.com/nzp-team/nzportable/assets/161973907/9bbe2c3f-c3fa-4667-988f-367f7560dabb)
![IMG_3542](https://github.com/nzp-team/nzportable/assets/161973907/522f78c7-0234-4097-9f08-9928883bd75d)

**Reply by framilano** [reply]
Nice, good job finding all the teddies! Well these bugs look more related to the game itself than the map, maybe @MotoLegacy could help us here.

Anyway, yes I used Trenchbroom, I suggest you to enter the [NZP discord](https://discord.gg/keqTsFmY), that's where I learnt everything and asked tons of questions. [Check the documentation](https://docs.nzp.gay/landing/#nazi-zombies-portable-documentation) and check this [tutorial series for Trenchbroom](https://www.youtube.com/playlist?list=PLgDKRPte5Y0AZ_K_PZbWbgBAEt5xf74aE) on the generic quake engine, most rules still apply here.

**Reply by ImTander** [reply]
I believe the inv link to the discord hit its max uses as it tells me its unavailable. Could I bother you for a fresh inv? Thanks for all your help!

**Reply by MotoLegacy** [reply]
https://discord.nzp.gay/

**Comment by MuzakyGood** [comment]
I found a bug in the PSP version of NZP and it only happens on this map, the Hellhounds Fog won't disappear after the round ends

**Reply by framilano** [reply]
That's something more related to NZP than the map itself, I guess. This map has some fog though, definitely not as much as during the dog round.

What do you think @MotoLegacy ?

**Reply by MotoLegacy** [reply]
Report it, thanks :)

**Comment by Revnova** [comment]
this is probably not something you can do. but this looks like a really great spot to have something peek out from around the corner and scare you when you least expect it xD

![fte-20240327023354-0](https://github.com/nzp-team/nzportable/assets/34180180/a7aa309d-8108-43df-a173-aeef320b3201)

**Reply by framilano** [reply]
It would be cool to have some kinda of random event like WW2 :')

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the download is not hosted on archive.org!

**Reply by framilano** [reply]
I just updated the download link!

**Comment by TheIdioticGamer** [comment]
Nice map bro, I made it to round 10, I absolutely love this map.

**Comment by actraiser30** [comment]
Amazing map great work 

There is another custom map probably one of the best "fru mansion"is it possible to convert that one?

https://www.moddb.com/games/call-of-duty-world-at-war/addons/nazi-zombie-frumansion

Thanks

**Reply by katniny** [reply]
if you're interested in remaking a map, please see documentation: https://docs.nzp.gay/ 

it's not fair to ask people to make maps for you when they're doing it for fun in their free time, so please stop

**Reply by actraiser30** [reply]
I don't have the hardware nor the experience or the time to make it otherwise il try to make it my self that I doubt I could make it.
Just a suggestion for a good map.

**Comment by Kingston4508** [comment]
Well done.  
![Z1](https://github.com/user-attachments/assets/1001c763-8eb0-4d01-82aa-7f23d10e82e0)
Took a bit of thinking to find the Teddy's.  2nd life.  Almost wanted to light off some fireworks and celebrate.  Great clues.  Great spots.  I stopped at 21 as I would not die.  The 5k, I was just thinking if any other maps had the same.  **Great idea!!!!**  Found this on my 3rd try.  Is there any other things that need to be found?  2 spots for endless points but I'll see if anyone else finds them.  Kind of makes the Z's, back and forth. All pile up then explode into greatness.  Sorry about the bad pic.  I'd love to play another map, if you will bless us with another.  Thanks again for this map.  Hi from Upstate NY.  

ps.

Send some rain if anyone has any, most of the lawn are almost white.

**Reply by framilano** [reply]
@Kingston4508 thank you! It was a one time project just to learn the basics of the engine, now I'm moving to Black Ops 3 modding :D
I think you pretty much found everything, maybe you didn't shot down all the explosive barrels? ;)
